tests/storage/sdm_copy_data_test.py

```diff
@@ -66,21 +66,20 @@
 from vdsm.storage import workarounds
 from vdsm.storage.sdm.api import copy_data
 
+BACKENDS = userstorage.load_config("storage.py").BACKENDS
 DEFAULT_SIZE = MiB
 
 
 @pytest.fixture(
     scope="module",
     params=[
-        userstorage.PATHS["mount-512"],
+        BACKENDS["mount-512"],
     ],
     ids=str,
 )
 def user_mount(request):
-    mount = request.param
-    if not mount.exists():
-        pytest.xfail("{} storage not available".format(mount.name))
-    return mount
+    with request.param:
+        yield request.param
```

Comment:
Previous code tried to make it more clear by using:

    mount = request.param

We can can do the same, or use:

    backend = request.param